ProjectSettings/QualitySettings.asset

```diff
@@ -149,7 +153,8 @@ QualitySettings:
     terrainBillboardStart: 50
     terrainFadeLength: 5
     terrainMaxTrees: 50
-    excludedTargetPlatforms: []
+    excludedTargetPlatforms:
```

Comment:
Which setting did you set and excluded Android?

Reply:
I left Android to only use the Low quality preset, so it is ignored for the rest